store/storetest/tokens_store.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package storetest
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/mattermost/mattermost-server/v6/model"
+	"github.com/mattermost/mattermost-server/v6/store"
+)
+
+func TestTokensStore(t *testing.T, ss store.Store) {
+	t.Run("TokensCleanup", func(t *testing.T) { testTokensCleanup(t, ss) })
+}
+
+func testTokensCleanup(t *testing.T, ss store.Store) {
+	now := model.GetMillis()
+
+	for i := 0; i < 10; i++ {
+		err := ss.Token().Save(&model.Token{
+			Token:    model.NewRandomString(model.TokenSize),
```

Comment:
Use `NewTestId` for this.

Reply:
Thanks for review! 
Why i could't use `model.NewRandomString`? This method is using by `NewToken` function.

Reply:
You definitely can. And in this case it's probably fine. The reason for `NewTestId` is that during tests, we have seen that randomly generated strings clash with user defined strings when searching for objects, leading to flaky tests. That's why `NewTestId` is specifically built to replace every odd character with a digit. In this case, it's okay to use this because we aren't searching by the string.